components/brave_education/common/education_urls.cc

```diff
@@ -0,0 +1,58 @@
+/* Copyright (c) 2024 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at https://mozilla.org/MPL/2.0/. */
+
+#include "brave/components/brave_education/common/education_urls.h"
+
+#include <string_view>
+#include <utility>
+
+#include "base/containers/fixed_flat_map.h"
+#include "brave/components/constants/webui_url_constants.h"
+
+namespace brave_education {
+
+namespace {
+
+constexpr std::string_view PageTypeToBrowserURL(EducationPageType page_type) {
+  switch (page_type) {
+    case EducationPageType::kGettingStarted:
+      return kBraveGettingStartedURL;
+  }
+}
+
+constexpr std::string_view PageTypeToServerURL(EducationPageType page_type) {
+  switch (page_type) {
+    case EducationPageType::kGettingStarted:
+      return "https://brave.com/getting-started/";
```

Comment:
The original spec with this design was reviewed [here](https://github.com/brave/reviews/issues/1561). This subdomain suggestion will require input from @timchilds and @bridiver.

Reply:
👍🏻 I've sent a message to @diracdeltas about this since she did the original spec review to see if this needs to be something we want to improve upon here since it would bring in additional work site side I'd suspect.

Reply:
No objection to isolating this to a subdomain; it would indeed let us restrict the CSP further.

Reply:
cc: @timchilds

Reply:
let's use welcome.brave.com or something along those lines

Reply:
Because this feature can be used for browser educational content other than just a welcome UX, I would suggest a more generic subdomain name, e.g. `browser-education.brave.com`. I don't have a strong opinion on the actual name.

Reply:
no strong opinions from me either. I'm good with `browser-education.brave.com` for the reason you suggested.